### PR TITLE
Updated to latest rules_kotlin 

### PR DESCRIPTION
**Pull request by sgowroji**
Updated to latest rules_kotlin

### CODE REVIEW

**Comment by sgowroji** [comment]
[dmaclach](https://github.com/dmaclach) This  fixes the protobuf failure at Bazel@HEAD. https://github.com/protocolbuffers/protobuf/issues/13829. Not familiar with the failure testcase. Can you have a look. Thanks!

**Comment by meteorcloudy** [comment]
@sgowroji I think the c++ code just happen to have similar names but it's in fact irrelevant to the breakage, you'll need to update rules_kotlin instead.

**Comment by sgowroji** [comment]
Oh okay. @meteorcloudy Thanks for information.

File: src/google/protobuf/compiler/objectivec/file.cc [hunk 1]
```diff
@@ -199,12 +199,12 @@ FileGenerator::CommonState::CollectMinimalFileDepsContainingExtensionsInternal(
         CollectMinimalFileDepsContainingExtensionsInternal(dep);
 
     // Everything the dep covered, this file will also cover.
-    transitive_deps_collector.insert(dep_info.transitive_compile_time_jars.begin(),
-                                     dep_info.transitive_compile_time_jars.end());
+    transitive_deps_collector.insert(dep_info.transitive_deps.begin(),
```
Comment (meteorcloudy) [inline src/google/protobuf/compiler/objectivec/file.cc:202]: Please revert this change.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: src/google/protobuf/compiler/objectivec/file.cc (hunk 1 above)
Comment (sgowroji) [reply src/google/protobuf/compiler/objectivec/file.cc:202]: Done.

**Comment by sgowroji** [comment]
Reverting this PR until there is release from rules_kotlin

**Comment by sgowroji** [comment]
@meteorcloudy Can you please review. Thanks!

**Review by meteorcloudy — APPROVED** [review]

**Comment by sgowroji** [comment]
Merged at cl/563772250

**Pull request closed without merging**